crds/clusterscan.yaml

```diff
@@ -45,6 +45,14 @@ spec:
               type: string
             retentionCount:
               type: integer
+            scanAlertRule:
+              nullable: true
+              properties:
+                alertOnComplete:
+                  type: boolean
+                alertOnFailure:
+                  type: boolean
```

Comment:
Should this just be one option? e.g. `alertOn:<complete|failure>`

`alertOnFailure` seems like a subset of `alertOnComplete`. Is there a use case where a user would want to turn both of them on? Seems like that would trigger two simultaneous alerts for complete + fail on a failed scan.

Reply:
Yes this is just to maintain parity with CIS v1 which allows to send out two alerts alertOnFailure and/or alertOnComplete.